webapp.yml

```diff
@@ -0,0 +1,71 @@
+# Description
+# ===========
+# This playbook create an Azure Linux Web app with Java container Tomcat 8.5,
+# then build a java hello world app, then ftp the war file to the Web app.
+#
+# Dependencies
+# ============
+# git
+# maven
+- hosts: localhost
+  connection: local
+  vars:
+    app_name: ansible_app1
+    location: eastus
+    linux_plan_name: linux_appplan1
+    repo_url: https://github.com/yungezz/helloworld.git
+    workspace: ~/helloworld
+
+  roles:
+    - Azure.azure_preview_modules
+  
+  tasks:
+  - name: Create resource group
+    azure_rm_resourcegroup:
+        name: "{{ resource_group_name }}"
+        location: "{{ location }}"
+
+  - name: Create a linux web app Java framework and Tomcat
```

Comment:
Create a Linux web app using Java framework and Tomcat

Reply:
with